azure-pipelines-e2e-tests-template.yml

```diff
@@ -57,7 +57,7 @@ stages:
             downloadPath: $(Build.ArtifactStagingDirectory)
 
         - pwsh: |
-            $framework = "netcoreapp3.1"
+            $framework = "net6.0"
```

Comment:
This change appears to be the cause of the backward-compatible test failures. A noddy fix can be attempted by replacing [`${framework}` in L75](https://github.com/dotnet/spark/pull/1112/files#diff-d94015cd9897fdda40a1fc70fee0200d9b9c9aa79807c120d3e7626577a5e4a3R75) with `netcoreapp3.1`, but I expect that will cause problems when backward-compatible versions targeting .NET 6.0 are finally published...

Reply:
I opened AFFogarty/spark#2 to trial this build/test fix, is anyone available to take a look at it?